docs/src/content/docs/reference/frontmatter.md

```diff
@@ -111,6 +111,18 @@ hero:
 ---
 ```
 
+To add support for a dark / light responsive hero image, modify the `hero.image` object like this, leaving other properties untouched.
```

Comment:
```suggestion
You can display different versions of the hero image in light and dark modes.
```

Following similar phrasing as https://starlight.astro.build/guides/customization/#light-and-dark-logo-variants